src/__tests__/production-exports.integrity.test.js

```diff
@@ -0,0 +1,97 @@
+/***************************************************************************************
+ * (c) 2025 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ ****************************************************************************************/
+
+describe(
```

Comment:
This test file is loaded first in the unit test suite and its job is to ensure that when we call the default export of these injectable modules that they don't throw any errors from our new auto validation function that checks we've injected the right amount of params for production.